travis-ci/vmtest/run.sh

```diff
@@ -243,7 +243,7 @@ create_rootfs_img() {
 download_rootfs() {
 	local rootfsversion="$1"
 	local dir="$2"
-	download "${PROJECT_NAME}-vmtest-rootfs-$rootfsversion.tar.zst" |
+	download "$(uname -m)/${PROJECT_NAME}-vmtest-rootfs-$rootfsversion.tar.zst" |
```

Comment:
instead of inlining `uname -m`, let's extract it into ARCH variable and use it consistently throughout all scripts?

Reply:
OK.

Reply:
What do you think about putting `ARCH=$(uname -m)` into `vmtest/helpers.sh`? An alternative option would be to require this variable to be set externally, but this would complicate running individual scripts.

Reply:
Yeah, helpers.sh is supposed to have a shared functionality between all scripts, so this makes sense.

Reply:
common.sh might be better name, though